### DISCUSSION TITLE
Size of card with and without extremas

### DISCUSSION THREAD

**Discussion by badewanne1234** [kalkih/mini-graph-card#1124] [Q&A] [answered]
So I have two cards, one with extremas shown and one without extremas.
The height of the two cards is different, the one without extremas is much smaller.
![grafik](https://github.com/user-attachments/assets/2a35ec63-0938-46ad-b81c-2e247a2c8c6e)

I can easily adjust the height with the `height` parameter in px, but that does not work with different screen sizes. 
Using % does not work at all. Any ideas how to make the size of the two cards identical on different screen sizes?

**Comment by ildar170975** [comment, marked as answer]
Use transparent color for extrema labels by card-mod. 
Ask for details in the main card-mod thread in Community

**Reply by badewanne1234** [reply]
Thank you, that did the trick. For anyone else looking for the solution, just add this to your card (it requireds also card-mod to be installed):

```
show:
  extrema: true
card_mod:
  style: |
    .info {
      color: transparent;
    }
```
